src/main/java/org/jabref/model/entry/identifier/SSRN.java

```diff
@@ -0,0 +1,78 @@
+package org.jabref.model.entry.identifier;
+
+import java.net.URI;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Represents an SSRN article, identified by its abstract ID.
+ */
+public class SSRN extends EprintIdentifier {
+
+    private static final String SSRN_URL_REGEX = "(https?://)?(papers\\.)?ssrn\\.com/(sol3/papers.cfm\\?)?abstract(_id)?=(?<id>\\d+)";
+    private static final Pattern SSRN_URL_FULL_MATCH = Pattern.compile("^" + SSRN_URL_REGEX + "$", Pattern.CASE_INSENSITIVE);
+    private static final Pattern SSRN_URL_MATCH = Pattern.compile(SSRN_URL_REGEX, Pattern.MULTILINE | Pattern.CASE_INSENSITIVE);
+
+    private final int abstractId;
```

Comment:
Store as `Integer` to avoid back and forth convertion from `int` to `Integer` and vice versa.